weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallPage/TraceCostStats.tsx

```diff
@@ -0,0 +1,105 @@
+import {formatTokenCost} from '@wandb/weave/util/llmTokenCosts';
```

Comment:
This is very similar to trace usage stats, but relies on LLMCostSchema, instead of usage